src/overlays/actors/ovl_Dm_Ravine/z_dm_ravine.h

```diff
@@ -5,9 +5,14 @@
 
 struct DmRavine;
 
+typedef void (*DmRavineActionFunc)(Actor*, GlobalContext*);
```

Comment:
Action func typedefs generally have the form (for example): `typedef void (*ArmsHookActionFunc)(struct ArmsHook*, GlobalContext*);`